src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/image.py

```diff
@@ -0,0 +1,50 @@
+from azure.cli.core.commands import LongRunningOperation
+from knack.util import CLIError
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import (
+    ImportImageParameters,
+    ImportSource
+)
+from ._utils import (
+    validate_managed_registry,
+    get_resource_id_by_registry_name
+)
+
+IMPORT_NOT_SUPPORTED = "Image imports are only supported for managed registries."
+
+def acr_image_import(cmd,
+                     client,
+                     registry_name,
+                     image=None,
+                     resource_id=None,
+                     source_image=None,
+                     target_tags=None,
+                     resource_group_name=None,
+                     untagged_target_repositories=None,
```

Comment:
`untagged_target_repository`, this parameter is too long to type. Think about shortening it. For example, server asks `osType` and in CLI we use `--os`.